src/components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Breadcrumbs, BreadcrumbItem } from '@nextui-org/react';
```

Comment:
As discussed , '@nextui-org/react' should not be used, instead the separate package should be used.